### DISCUSSION TITLE
Sorting tasks in To-Do

### DISCUSSION THREAD

**Discussion by plegrand1** [gsantner/markor#1646] [Q&A] [answered]
Markor version v2.8.6(130)
Android 11
MIUI 12.5.1
XIAOMI MI 9

When creating a new task in To-Do, while sorting by "Description (ascending)" is activated, once the task is saved and when switching back to display mode the display order is not respected automatically.
In the edit mode, you have to select the "Description (ascending)" sort order again (which is already selected) to re-sort all the tasks.

Is this a bug?

If not, is it possible to make sure that the selected display order is systematically respected (at refresh for example)?

Thanks again for your great application Pascal

Translated with www.DeepL.com/Translator (free version)

> Do you mean

Yes, I mean that new tasks are not automatically re-ordered
Thanks

**Comment by gsantner** [comment, marked as answer]
Do you mean

a. View mode text is not in sync with what you have in edit mode (like missing text)
b. or is your description about that, that new tasks are not automatically re-ordered?

----

The buttons on the bottom are to apply action on the text. The sorting is done on the textfile when you click, one time, not continously.

For a different, filtered/sorted view, click on the filename/title on the top left.

**Comment by harshad1** [comment]
@plegrand1 Long pressing the sort button will apply the last sort method used. So for example, you can make a bunch of edits and then long press the button again for a relatively quick way to keep your list sorted.
